apps/mobile/package.json

```diff
@@ -21,6 +21,7 @@
     "bdk-rn": "^0.30.0",
     "bip39": "^3.1.0",
     "d3": "^7.9.0",
+    "d3-sankey": "^0.12.3",
```

Comment:
Let's add this dependency in the docs page `apps/docs/pages/getting-started/dependencies.mdx`